source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -49,6 +49,7 @@
 
 
 CAR_PARK_DURATION = 300  # secs
+allowed_physical_mode_for_transfert_path = ['Bus', 'Tramway', 'RapidTransit']
```

Comment:
```suggestion
allowed_physical_mode_for_transfert_path = ('Bus', 'Tramway', 'RapidTransit')
```
Tuple is non mutable in python

Reply:
Ok thanks for the tips